besca/pl/_nomenclature_network.py

```diff
@@ -0,0 +1,33 @@
+from matplotlib import pyplot as plt
+import pandas as pd
+_nomenclature_network.import importlib
+
+networkx_import = importlib.util.find_spec('networkx')
```

Comment:
Would this create an issue when loading besca if networkx or pydot is absent? 
I would have assume it should be tried when nomenclature_nework is used.

Reply:
(ie. Installing Besca would work but importing besca would fail)

Reply:
This does not create an issue beacause I found out that `networkx` is actually installed by another besca dependency.